### PR TITLE
Sean j unit add cmd

### PR DESCRIPTION
**Pull request by SeanHoWB**
Added add cmd JUnit tests and placed parser in a try catch

### CODE REVIEW

File: src/test/java/seedu/mindmymoney/command/AddCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    @Test
+    void addCommand_nonIntAmount_expectOriginalList() {
+        String inputString = "expenditure deadbeef";
+        ArrayList<Expenditure> testList = Lists.expenditures;
+        new AddCommand(inputString);
+        assertEquals(Lists.expenditures, testList);
+    }
+
+}
```
Comment (khseah) [inline src/test/java/seedu/mindmymoney/command/AddCommandTest.java:38]: Maybe just add a newline so github wont complain

File: src/test/java/seedu/mindmymoney/command/AddCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
+package seedu.mindmymoney.command;
```
Comment (khseah) [inline src/test/java/seedu/mindmymoney/command/AddCommandTest.java:1]: I think there is a folder to write all the JUnit tests. src > test > java > seedu.mindmymoney > MindMyMoneyTest. Maybe can add there instead? Or create a command folder similar to the one in main and add the AddCommandTest there.

**Review by khseah — COMMENTED** [review]
Good idea for putting the main logic of parser in a try-catch

File: src/test/java/seedu/mindmymoney/command/AddCommandTest.java (hunk 2 above)
Comment (SeanHoWB) [reply src/test/java/seedu/mindmymoney/command/AddCommandTest.java:1]: This one is in the command folder, its when i follow lecture video and right click generate test function
image

Comment (khseah) [reply src/test/java/seedu/mindmymoney/command/AddCommandTest.java:1]: Ahh I've missed that, my bad. Yup, that everything looks good!

**Pull request merged**